src/stories/Quiz/Component/QuizResults/QuizResultsDecorator.tsx

```diff
@@ -37,3 +37,17 @@ export function QuizResultsPrimaryDecorator(Story: any) {
     </div>
   );
 }
+
+export function QuizResultsWithSummaryDecorator(Story: any) {
+  const contextValue = useMockContextValue(undefined, { withSummary: true });
+
+  return (
+    <div className='cio-quiz'>
+      <QuizContext.Provider value={contextValue}>
+        <div>
+          <Story Component={Story} />
```

Comment:
<StoryPreview /> Instead of stories for all window sizes

Reply:
[cbb7699](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/168/commits/cbb7699d030b8b5ea880dfc70c4b74c58716ceec)